config/locales/en.yml

```diff
@@ -3,6 +3,7 @@ en:
   opensearch_description: "Stanford Libraries"
   catalog_search:
     display_name: "SearchWorks catalog"
+    sub_heading: Books, media, physical &amp; digital resources
```

Comment:
This looks like it's not getting an `html_safe` in your screenshot

Reply:
Good catch. Fixed.